Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<metadata>
```

Comment:
Meta file is incorrect

Reply:
I think I will add the content manually because I believe its a bug

Reply:
Done